backend/src/main/resources/db/migration/local/V1_1__RoleOperations.sql

```diff
@@ -1,4 +1,4 @@
-INSERT INTO role (name) values ('Vasco');
+INSERT INTO role (name) values ('Frontend Developer');
```

Comment:
Take caution with this change, after rebase use the "real" data which is already in main branch.